### PR TITLE
Created travis.yml file

### PR DESCRIPTION
**Pull request by rajesh-ibm-power**
Created travis.yml file for jetty-reactive-httpclient package

### CODE REVIEW

File: travis-ymls/jetty-reactive-httpclient.travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+dist: xenial
+
+arch:
+  - amd64
+  - ppc64le
+
+before_install:
+  - export M2_HOME=$HOME/apache-maven-3.5.4
+  - if [ ! -d $M2_HOME/bin ]; then curl https://archive.apache.org/dist/maven/maven-3/3.5.4/binaries/apache-maven-3.5.4-bin.tar.gz | tar zxf - -C $HOME; fi
+  - export PATH=$M2_HOME/bin:$PATH
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/jetty-reactive-httpclient.travis.yml:26]: We have maven pre- installed in ppc64le image. Please remove this installation. @rajesh-ibm-power

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

**Comment by gururajrkatti** [comment]
@rajesh-ibm-power any updates here ?

**Comment by gururajrkatti** [comment]
@rajesh-ibm-power any updates on this ?

**Comment by rajesh-ibm-power** [comment]
@gururajrkatti updated dist to bionic .kindly verify

**Review by gururajrkatti — COMMENTED** [review]
Looks good to me. @Siddhesh-Ghadi I have verified that maven installation has been removed. Do you have any other observations/comments ?

**Comment by Siddhesh-Ghadi** [comment]
No other changes. @gururajrkatti

**Pull request merged**